core/src/components/range/range.md.vars.scss

```diff
@@ -19,20 +19,14 @@ $range-md-hit-width:                         42px !default;
 $range-md-hit-height:                        $range-md-slider-height !default;
 
 /// @prop - Height of the range bar
-$range-md-bar-height:                        2px !default;
+$range-md-bar-height:                        4px !default;
 
 /// @prop - Background of the range bar
 $range-md-bar-background-color:              $background-color-step-250 !default;
 
 /// @prop - Background of the active range bar
 $range-md-bar-active-background-color:       current-color(base) !default;
 
-/// @prop - Width of the range knob
```

Comment:
These variables are unused in the implementation. When the `--knob-size` CSS variable was introduced these were not removed.